src/data/database.js

```diff
@@ -417,6 +423,7 @@ class Database {
      */
   async createPayerQuoteParty (txn, quoteId, party, amount, currency) {
     // note amount is negative for payee and positive for payer
+    // TODO: enums needed
```

Comment:
Would you add there to an enum and use it in the function call as it it used in many places

Reply:
Ok, let me look at cs-shared to see if there is an enum I can use here.

Reply:
please update these with enums